lib/project_types/node/commands/create.rb

```diff
@@ -2,6 +2,7 @@
 module Node
   module Commands
     class Create < ShopifyCli::SubCommand
+      LOCAL_DEBUG = 'SHOPIFY_APP_CLI_LOCAL_PARTNERS'
```

Comment:
Rather than re-declaring this constant, we can reference the existing one, e.g. `ShopifyCli::PartnersApi::LOCAL_DEBUG` ?